addons/stock/models/stock_quant.py

```diff
@@ -122,8 +128,31 @@ def _compute_inventory_date(self):
     @api.depends('inventory_quantity')
     def _compute_inventory_diff_quantity(self):
         for quant in self:
+            quant.inventory_quantity_set = True
             quant.inventory_diff_quantity = quant.inventory_quantity - quant.quantity
 
+    @api.depends('inventory_quantity', 'quantity', 'product_id')
+    def _compute_is_outdated(self):
+        for quant in self:
+            quant.outdated = False
+            if quant.product_id and float_compare(quant.inventory_quantity - quant.inventory_diff_quantity, quant.quantity, precision_rounding=quant.product_uom_id.rounding) and quant.inventory_quantity_set:
+                quant.outdated = True
```

Comment:
That would be safer to modify the compute's name :suspect: 
```suggestion
            quant.is_outdated = False
            if quant.product_id and float_compare(quant.inventory_quantity - quant.inventory_diff_quantity, quant.quantity, precision_rounding=quant.product_uom_id.rounding) and quant.inventory_quantity_set:
                quant.is_outdated = True
```